test/unit/org/apache/cassandra/tools/nodetool/RingTest.java

```diff
@@ -57,23 +59,27 @@ public static void setup() throws Exception
     @Test
     public void testRingOutput()
     {
-        final HostStatWithPort host = new HostStatWithPort(null, FBUtilities.getBroadcastAddressAndPort(),
-                                                           false, null);
+        final HostStatWithPort host = new HostStatWithPort(null,
+                                                           FBUtilities.getBroadcastAddressAndPort(),
+                                                           false,
+                                                           null);
         validateRingOutput(host.ipOrDns(false),
-                            "ring");
-        Arrays.asList("-pp", "--print-port").forEach(arg -> {
-            validateRingOutput(host.ipOrDns(true),
-                               "-pp", "ring");
-        });
-
-        final HostStatWithPort hostResolved = new HostStatWithPort(null, FBUtilities.getBroadcastAddressAndPort(),
-                                                                   true, null);
-        Arrays.asList("-r", "--resolve-ip").forEach(arg -> {
-            validateRingOutput(hostResolved.ipOrDns(false),
-                               "ring", "-r");
-        });
+                           "ring");
+        Arrays.asList("-pp", "--print-port").forEach(arg -> validateRingOutput(host.ipOrDns(true),
+                                                                               "-pp",
+                                                                               "ring"));
```

Comment:
Nit: I think we don't need to split this line
```suggestion
        Arrays.asList("-pp", "--print-port").forEach(arg -> validateRingOutput(host.ipOrDns(true), "-pp", "ring"));
```